scripts/htmlproofer

```diff
@@ -8,11 +8,23 @@ require 'jekyll'
 retries = 0
 retry_delay = 0
 
+
```

Comment:
We may not need these changes, since `--url-ignore` can be passed multiple times to ignore multiple URLs:

```
--url-ignore congress.gov --url-ignore lastpass.com
```